src/main/scala/tech/cryptonomic/nautilus/metering/repositories/InfluxDbRepo.scala

```diff
@@ -27,22 +27,22 @@ object Models {
 trait InfluxDbRepo {
 
   /** Reads amount of queries grouped by 5minute intervals */
-  def get5minQueries(apiKeys: List[String]): Future[List[QueryCQ]]
+  def get5minQueries(apiKeys: List[String], from: Option[Long]): Future[List[QueryCQ]]
```

Comment:
Can we have docs on all of the following parameters.
What is it representing, using which time unit, and relative to now?

Reply:
ok, I've added comments in the implementation